src/core/outlet_set.ts

```diff
@@ -41,7 +41,11 @@ export class OutletSet {
 
   getSelectorForOutletName(outletName: string) {
     const attributeName = this.schema.outletAttributeForScope(this.identifier, outletName)
-    return this.controllerElement.getAttribute(attributeName)
+    const selector = this.controllerElement.getAttribute(attributeName)
+
+    if (selector) return selector
+
+    return this.scope.controllerSelector
```

Comment:
This is minor, but what do you think about turning this early/middle `return` into either a full conditional, a ternary, or maybe `??`?

```ts
if (selector) return selector
else return this.scope.controllerSelector

if (selector) {
  return selector
} else {
  return this.scope.controllerSelector
}

return selector ?
  selector :
  this.scope.controllerSelector

return selector ?? this.scope.controllerSelector
```

Reply:
I like the idea of using the nullish coalescing operator here

Reply:
I went with ` ||`, thanks!